level1c4pps/vgac2pps_lib.py

```diff
@@ -70,84 +72,288 @@
+        },
+        "tb37_night": {
+            "viirs_channel": "M12",
+            "slope": 0.9967,
+            "offset": 0.85,
+            "min_sunzenith": 89,
+            "comment": "based on nadir collocation data for SZA >= 89",
+        },
+        "tb37_day": {
+            "viirs_channel": "M12",
+            "slope": 0.9455,
+            "offset": 12.69,
+            "max_sunzenith": 80,
+            "comment": "based on NASA SBAFS for VZA < 5 and SZA < 80",
+        },
+        "tb37_twilight": {
+            "viirs_channel": "M12",
+            "slope": 0.9711,
+            "offset": 6.77,
+            "min_sunzenith": 80,
+            "max_sunzenith": 89,
+            "comment": "The linear average of the SBAFs for t37_day and t37_night. 80<= SZA <89",
+        },
+        "tb11": {
+            "viirs_channel": "M15",
+            "slope": 1.002,
+            "offset": -0.4,
+            "comment": "based on nadir collocation data for SZA 0 -180",
+        },
+        "tb12": {
+            "viirs_channel": "M16",
+            "slope": 0.9934,
+            "offset": 1.52,
+            "comment": "based on nadir collocation data for SZA 0-180",
+        },
+    },
+    "v6": {
+        "r06": {
+            "viirs_channel": "M05",
+            "slope": 0.9393,
+            "offset": 0.57,
+            "comment": "Based on collocation data, VZA < 3 for SZA < 60",
+        },
+        "r09": {
+            "viirs_channel": "M07",
+            "slope": 0.9104,
+            "offset": -0.90,
+            "comment": "Based on collocation data, VZA < 3 for SZA < 60",
+        },
+        "tb37_night": {
+            "viirs_channel": "M12",
+            "slope": 0.9967,
+            "offset": 0.85,
+            "min_sunzenith": 89,
+            "comment": "Based on collocations data for SZA >= 89",
+        },
+        "tb37_day": {
+            "viirs_channel": "M12",
+            "slope": 0.9455,
+            "offset": 12.69,
+            "max_sunzenith": 80,
+            "comment": "Based on collocations for SZA < 80 o and VZA < 5",
+        },
+        "tb37_twilight": {
+            "viirs_channel": "M12",
+            "slope": 0.9711,
+            "offset": 6.77,
+            "min_sunzenith": 80,
+            "max_sunzenith": 89,
+            "comment": "The linear average of the SBAFs for t37_day and t37_night. 80<= SZA <89",
+        },
+        "tb11": {
+            "viirs_channel": "M15",
+            "slope": 1.003,
+            "offset": -0.77,
+            "comment": "Based on collocation data for VZA < 3 and SZA 0 -180",
+        },
+        "tb12": {
+            "viirs_channel": "M16",
+            "slope": 1.002,
+            "offset": -0.69,
+            "comment": "Based on collocation data for VZA < 3 and SZA 0-180",
+        }
+    }
+}
+
+
+def convert_to_noaa19(scene, sbaf_version):
+    """_applies AVHRR SBAF to VGAC channels_
+
+    Args:
+        scene (_type_): _description_
```

Comment:
don't forget to add description, or remove it.

Reply:
changed to  """ Applies AVHRR SBAF to VGAC channels.""" to be consistent with the rest of the code